index.js

```diff
@@ -27,4 +29,32 @@ form.addEventListener('submit', (e) => {
     e.preventDefault();
     errorMsg.classList.add('display-content');
   }
+});
+
+// Enable Local storage for the form
+
+function populateStorage() {
+  const nameValue = userName.value;
+  const emailValue = email.value;
+  const userMessage = messageContent.value;
+
+  const userInputs = {
+    name: nameValue,
+    email: emailValue,
+    message: userMessage,
+  };
+
+  localStorage.setItem(userInputs, JSON.stringify(userInputs));
+}
+
+const parsedInputs = JSON.parse(localStorage.getItem('userInputs'));
```

Comment:
the inputs data are stored correctly but when I refresh the page the data doesn't fill the form automatically so please try to fix that so when a user reload the page should the previous data be there 
![image](https://user-images.githubusercontent.com/61790587/169102937-de90266d-5f1b-4d2f-8ec9-d22ac3dd6783.png)